README.md

```diff
@@ -51,7 +54,8 @@ for presentational data, like rendered map tiles.
 
 One MBTiles file represents a single tileset, optionally including grids
 of interactivity data. Multiple tilesets - layers, or maps in other terms,
-can be represented by multiple MBTiles files.
+can be represented by multiple MBTiles files. Vector tiles within a single
+MBTiles file can contain multiple layers.
```

Comment:
Good point. I added the sentence only because the presence of multiple layers in the vector tile part seemed to contradict the statement here that there is only a single layer. I'll think of a better way to word this.